Cargo.toml

```diff
@@ -73,7 +73,7 @@ serde_json = "1.0.58"
 clap = { version = "3.1", features = ["derive", "wrap_help"], optional = true }
 subprocess = "0.2.6"
 termcolor = "1.1.0"
-toml_edit = { version = "0.13.3", features = ["easy"] }
+toml_edit = { version = "0.14.4", features = ["easy"] }
```

Comment:
Please specify the `perf` feature as well.

Reply:
oops, sorry! I should have looked at the changelog. Done :+1: